user_guide_src/source/incoming/filters.rst

```diff
@@ -176,6 +176,10 @@ specify the method name in lowercase. It's value would be an array of filters to
 In addition to the standard HTTP methods, this also supports one special case: 'cli'. The 'cli' method would apply to
 all requests that were run from the command line.
 
+.. Warning:: If you use ``$methods`` filters, you should :ref:`disable auto-routing <use-defined-routes-only>`.
+    Because auto-routing permits any HTTP method to access a controller.
```

Comment:
```suggestion
.. Warning:: If you use ``$methods`` filters, you should :ref:`disable auto-routing <use-defined-routes-only>`
    because auto-routing permits any HTTP method to access a controller.
```